src/_temp_types/api/teams.ts

```diff
@@ -0,0 +1,28 @@
+export type ApiProjectSet = {
+    id: number
+    name: string
+    teams: ApiProject[]
+    createdAt: string
+    updatedAt: string
+}
+
+export type ApiProject = {
+    id: number
+    name: string
+    members: ApiCourseMember[] // This is supposed to be empty
+    createdAt: string
+    updatedAt: string
+    slug: string
+    requirements: ApiProjectRequirement[]
+    max_people: number
+    min_people: number
+    team_set: number
```

Comment:
Functionally, this is the id of a project set. It either (a) has no value here or (b) should be named better